tests/foreman/cli/test_product.py

```diff
@@ -467,3 +469,31 @@ def test_positive_sync_by_label(self):
             'organization-id': org['id'],
         })
         self.assertEqual(u'Syncing Complete.', product['sync-state'])
+
+    @tier2
+    def test_positive_package_count(self):
+        """Check that packages count is correctly filtered by product id
+
+        :id: 151f60a3-0b94-4658-8b0d-0d022f4f1d8f
+
+        :expectedresults: Packages only from synced product returned
+
+        :BZ: 1422552
+
+        :CaseLevel: Integration
+        """
+        org = make_org()
+        for _ in range(3):
+            product = make_product({'organization-id': org['id']})
+            repo = make_repository({'product-id': product['id']})
+            Product.synchronize({
+                'id': product['id'],
+                'organization-id': org['id'],
+            })
+            packages = Package.list({'product-id': product['id']})
+            repo = Repository.info({'id': repo['id']})
+            self.assertEqual(
+                int(repo['content-counts']['packages']),
+                len(packages)
+            )
+            self.assertEqual(len(packages), 32)
```

Comment:
please declare and use constants FAKE_1_YUM_REPO_PACKAGES_COUNT as we have already done with errata count